package.json

```diff
@@ -54,8 +54,8 @@
         "size-limit": "^6.0.3",
         "tslint": "^6.1.3",
         "typescript": "^4.5.4",
-        "webpack": "4.35.0",
-        "webpack-cli": "3.3.5",
+        "webpack": "^5.72.0",
+        "webpack-cli": "^4.9.2",
         "webpack-dev-server": "3.7.2"
     },
     "size-limit": [
```

Comment:
How will these bundles be deployed? And how will their versioning be done?

Reply:
Similar to how supertokens-website works 

```
https://cdn.jsdelivr.net/gh/supertokens/supertokens-web-js@0.0/bundle/emailpassword.js
```

^ To use email password from version 0.0.X for example

Reply:
The link uses gh for now because to test I was using them from github, it will be an npm link when we release